src/controllers/LayoutController.ts

```diff
@@ -121,36 +119,33 @@ export class LayoutController {
 
     /**
      * This method will set the height of the layout to a specific value
+     * @param layoutId The ID of a specific layout
      * @param value The string value that will be calculated (f.e. 1+1 will reult in 2) The notation is in pixels
```

Comment:
same as @alexandraFlavia9  already mentioned applies for this file as well

Reply:
done e011559959dd20a563d03bc25ee96d06b7df5991